maps/torch/items/clothing/terran-accessory.dm

```diff
@@ -14,73 +14,69 @@ ranks - TCC
+	name = "ranks (E-7 petty officer first class)"
+	desc = "Insignia denoting the rank of Chief Petty Officer."
 
 /obj/item/clothing/accessory/terran/rank/navy/enlisted/e9
-	name = "ranks (E-9 michman)"
-	desc = "Insignia denoting the rank of Michman."
-
-/obj/item/clothing/accessory/terran/rank/navy/enlisted/e9_alt1
-	name = "ranks (E-9 michman of the independent navy)"
-	desc = "Insignia denoting the rank of Michman of the Independent Navy."
+	name = "ranks (E-9 senior chief petty officer)"
+	desc = "Insignia denoting the rank of Senior Chief Petty Officer."
 
 /obj/item/clothing/accessory/terran/rank/navy/officer
 	name = "ranks (O-1 ensign)"
 	desc = "Insignia denoting the rank of Ensign."
 	icon_state = "terranrank_officer"
 
 /obj/item/clothing/accessory/terran/rank/navy/officer/o2
-	name = "ranks (O-2 leytenant)"
-	desc = "Insignia denoting the rank of Leytenant."
+	name = "ranks (O-2 lieutenant)"
+	desc = "Insignia denoting the rank of Lieutenant."
 
 /obj/item/clothing/accessory/terran/rank/navy/officer/o3
-	name = "ranks (O-3 starshy leytenant)"
-	desc = "Insignia denoting the rank of Starshy Leytenant."
+	name = "ranks (O-3 senior lieutenant)"
+	desc = "Insignia denoting the rank of Senior Lieutenant."
 
 /obj/item/clothing/accessory/terran/rank/navy/officer/o4
-	name = "ranks (O-4 corvette-komandor)"
-	desc = "Insignia denoting the rank of Corvette-Komandor."
+	name = "ranks (O-4 corvette commander)"
+	desc = "Insignia denoting the rank of Corvette Commander."
 
 /obj/item/clothing/accessory/terran/rank/navy/officer/o5
-	name = "ranks (O-5 komandor)"
-	desc = "Insignia denoting the rank of Komandor."
+	name = "ranks (O-5 commander)"
+	desc = "Insignia denoting the rank of Commander."
 
 /obj/item/clothing/accessory/terran/rank/navy/officer/o6
-	name = "ranks (O-6 kapitan)"
-	desc = "Insignia denoting the rank of Kapitan."
+	name = "ranks (O-6 captain)"
+	desc = "Insignia denoting the rank of Captain."
 	icon_state = "terranrank_command"
 
 /obj/item/clothing/accessory/terran/rank/navy/flag
-	name = "ranks (O-7 kontradmiral)"
-	desc = "Insignia denoting the rank of Kontradmiral."
+	name = "ranks (O-7 rear admiral)"
+	desc = "Insignia denoting the rank of Rear Admiral."
 	icon_state = "terranrank_command"
 
 /obj/item/clothing/accessory/terran/rank/navy/flag/o8
-	name = "ranks (O-8 wiceadmiral)"
-	desc = "Insignia denoting the rank of Wiceadmiral."
+	name = "ranks (O-8 vice admiral)"
+	desc = "Insignia denoting the rank of Vice Admiral."
 
 /obj/item/clothing/accessory/terran/rank/navy/flag/o9
 	name = "ranks (O-9 admiral)"
 	desc = "Insignia denoting the rank of Admiral."
 
 /obj/item/clothing/accessory/terran/rank/navy/flag/o10
-	name = "ranks (O-10 admiral of the independent navy)"
-	desc = "Insignia denoting the rank of Admiral of the Independent Navy."
\ No newline at end of file
+	name = "ranks (O-10 master admiral of the humanity)"
+	desc = "Insignia denoting the rank of Master Admiral of the Humanity."
```

Comment:
While most other stuff is, uh, subject to Rowtree, I can't help but think `Master Admiral of Humanity` would be better.

Reply:
Ah, the famous English grammar? I'll change it.